docs/usage/user-stories/swissquote.md

```diff
@@ -0,0 +1,235 @@
+
+1. The newer OS version prevents you from installing an outdated runtime
+1. You need to find a compatible runtime for your new hardware
+1. Your app’s dependencies aren’t compatible with your new runtime, you need to update them as well
+
+Instead of a single change, you now have a combination of changes, each can go wrong in a different way.
+
+### A new vulnerability is found on a library you depend on
+
+As we’ve covered at the beginning of the article this can happen and can be an all-hands-on-deck kind of situation.
+Imagine a legacy application; continuous integration is constantly failing — or worse it may not even exist.
+How long do you think it will take to deploy that single library update?
+
+### The team already has too many dependencies in too many different versions
+
+That’s unfortunate, but it will happen if your team or company exists for long enough.
+Upgrading libraries is not only about bumping versions, sometimes it’s also about replacing a library by another library.
+
+## How do you see your dependencies?
+
+Look at your software as if it was a train.
+You are the locomotive and each wagon is a dependency, how long would your train be?
+Locomotives can pull dozens, even hundreds of wagons, but they eventually reach a limit.
+The situation is the same for your brain, each dependency adds to your cognitive load, at some point it’s just too much.
+You are using three different libraries for caching?
+There is a chance you need to know how each library works, and your present and future teammates might have to know that too.
+
+Which of these two would you prefer to start with:
+
+- A project that is fairly up-to-date where you can bump the dependency, run the deployment pipeline and go back to your day
+- A project that wasn’t touched in years; every dependency is outdated and Continuous Integration is red on all branches, if running at all
+
+Am I exaggerating with my examples?
+Maybe a little, but I have seen cases very close to those.
+
+Let me ask you some questions about your projects:
+
+- How many times did you create a project and never upgrade its dependencies?
+- How many times did you have to get back to an old project, and had to use a new library but couldn’t because there is another library at an old version that isn’t compatible?
+- How fast can you upgrade a single dependency on all your applications?
+
+As time goes forward, projects come and go, you will most likely have decade-old, and recent ones.
+Some might be using the latest version of Java with the latest Spring, some with slightly outdated libraries and others might be using Stripes and libraries that have had no release in 7 years.
+
+Every company that’s been around for sometime has that old project still running.
+People talk about it laughingly but become livid when a request comes to change anything in it.
+
+![A person sweating profusely at the idea of having to touch an old project](../assets/images/swissquote_sweating_guy.jpg)
```

Comment:
There's a superfluous white line under the image. As documentation also has a dark mode, it'd be nicer if we do without it.

Reply:
Oh I didn't notice that, done